packages/actions/scripts/prepare-release.mjs

```diff
@@ -14,6 +14,9 @@ async function main() {
     const packageJson = JSON.parse(await fs.readFile('package.json', 'utf-8'));
     const apiVersion = packageJson.apiVersion;
     const sdkVersion = packageJson.version;
+    const prId = process.env.CI_PR_ID;
```

Comment:
@brapoprod this script runs on every PR build. 

The problem here is that I updated the files, but if another PR targeting the same SDK version is created/updated a new build will happen and overwrite my version (modified) of the actions with their version (unmodified). 

As a workaround for now I added this, so that the files are added in `.../prs/462/...` as well, so at least QA and I have a stable reference to my version of the files. 

Any thoughts?